### PR TITLE
EN-4684 Elastic search improve (log more infromation)

### PR DESCRIPTION
**Pull request by miiu96**
Log more information from elastic search.
Fix problem with goroutine in for instruction.

### CODE REVIEW

File: core/indexer/elasticsearch.go [hunk 1]
```diff
@@ -321,15 +322,17 @@ func (ei *elasticIndexer) saveShardValidatorsPubKeys(shardId uint32, shardValida
 	}
 
 	res, err := req.Do(context.Background(), ei.db)
+	ei.logger.Info(fmt.Sprintf("Response validators public key elastic indexer %v", res))
+
 	if err != nil {
-		ei.logger.Warn(fmt.Sprintf("Could not index validators public keys: %s", err))
+		ei.logger.Info(fmt.Sprintf("Could not index validators public keys: %s", err))
```
Comment (ccorcoveanu) [inline core/indexer/elasticsearch.go]: Same question as above ^

File: core/indexer/elasticsearch.go [hunk 2]
```diff
@@ -321,15 +322,17 @@ func (ei *elasticIndexer) saveShardValidatorsPubKeys(shardId uint32, shardValida
...
 		return
 	}
 
 	defer closeESResponseBody(res)
 
 	if res.IsError() {
-		ei.logger.Warn(res.String())
+		ei.logger.Info(res.String())
```
Comment (ccorcoveanu) [inline core/indexer/elasticsearch.go]: Same question as above ^

File: core/indexer/elasticsearch.go [hunk 3]
```diff
@@ -306,7 +307,7 @@ func (ei *elasticIndexer) saveShardValidatorsPubKeys(shardId uint32, shardValida
 	shardValPubKeys := ValidatorsPublicKeys{PublicKeys: shardValidatorsPubKeys}
 	marshalizedValidatorPubKeys, err := ei.marshalizer.Marshal(shardValPubKeys)
 	if err != nil {
-		ei.logger.Warn("could not marshal validators public keys")
+		ei.logger.Info("could not marshal validators public keys")
```
[the remark this answers is not in the document]
Comment (miiu96) [reply core/indexer/elasticsearch.go]: Changed. Now it a warning.

File: core/indexer/elasticsearch.go [hunk 4]
```diff
@@ -321,15 +322,17 @@ func (ei *elasticIndexer) saveShardValidatorsPubKeys(shardId uint32, shardValida
 	}
 
 	res, err := req.Do(context.Background(), ei.db)
+	ei.logger.Info(fmt.Sprintf("Response validators public key elastic indexer %v", res))
```
[the remark this answers is not in the document]
Comment (miiu96) [reply core/indexer/elasticsearch.go]: It works with res. But i will change it with your suggestion.

File: core/indexer/elasticsearch.go (hunk 1 above)
Comment (miiu96) [reply core/indexer/elasticsearch.go]: Changed.

File: core/indexer/elasticsearch.go (hunk 2 above)
Comment (miiu96) [reply core/indexer/elasticsearch.go]: Changed.

**Pull request merged**